gap/congruences/conglatt.gi

```diff
@@ -510,14 +500,15 @@ InstallMethod(DotString,
 "for a congruence poset and a record",
 [IsCongruencePoset, IsRecord],
 function(poset, opts)
-  local congs, S, symbols, i, nr, rel, str, j, k;
+  local out_nbs, congs, S, symbols, i, nr, rel, str, j, k;
+  out_nbs := OutNeighbours(poset);
```

Comment:
Dont need this line. Throughout this method, change `Length(out_nbs)` and `Length(rel)` to `Size(poset)`.